ocicrypt-rs/src/blockcipher/mod.rs

```diff
@@ -12,7 +12,7 @@ use serde::{de, Deserialize, Deserializer, Serialize, Serializer};
 mod aes_ctr;
 use aes_ctr::AESCTRBlockCipher;
 
-mod rand;
+pub mod rand;
```

Comment:
we are getting ready with the work. @arronwy can you comment on this part. looks OK?

Reply:
yes, it's ok from my side.